### PR TITLE
Add Linux instructions to docs/README.md

### PR DESCRIPTION
**Pull request by wilsonianb**

### CODE REVIEW

**Comment by wilsonianb** [comment]
I'm wondering if we should rename `Linux` to `Docker` since these instructions also work on Windows and MacOS.

File: docs/README.md [hunk 1]
```diff
@@ -29,7 +35,11 @@ Under construction
 
 ### Linux
 
-Under construction
+1. Install [Docker](https://docs.docker.com/engine/installation/)
+2. Build Docker image. From the rippled root folder:
+```
+docker build -t rippled-docs docs/
```
Comment (seelabs) [inline docs/README.md]: I followed the instructions to install docker. If I run the command above I get the following message: `Cannot connect to the Docker daemon. Is the docker daemon running on this host?`. However, if I use `sudo` the above appears to work (it's still running). Is this expected? If so, we should document that `sudo` is required. If not we should understand what I'm doing wrong.

**Comment by seelabs** [comment]
Very nice. I'll wait to give a thumbs up until we get the `sudo` issue worked out, but other than that this looks good.

File: docs/README.md (hunk 1 above)
Comment (wilsonianb) [inline docs/README.md]: Added `sudo` to the instructions

**Comment by seelabs** [comment]
:+1:

**Comment by MarkusTeufelberger** [comment]
The currenct boost version is 1.62 and the download via plain http without any integrity check afterwards seems a bit risky. Also the Ubuntu version should be fixed (or at least working versions documented) instead of "latest" or this risks breaking in the future.

**Comment by wilsonianb** [comment]
@MarkusTeufelberger latest commit should address those issues. Thanks

**Comment by wilsonianb** [comment]
Sorry, looks like Boost 1.62.0 needs some extra dependencies. Fix is in the works.

**Comment by wilsonianb** [comment]
squashed and rebased on 0.40.0-b9

**Comment by seelabs** [comment]
I had some trouble re-running the tests. Giving docker the `--no-cache` flag fixed my issues (thanks @wilsonianb for the help). Do we need to document that?

At any rate, tests run and still :+1:

**Comment by wilsonianb** [comment]
I don't think the instructions should have `--no-cache`, but we can reassess if problems are reported or we re-encounter your issue with future changes to the Dockerfile.

**Comment by nbougalis** [comment]
Merged as 47adc728db2f42a20db52b7f5852b83c4e88d187

**Pull request closed without merging**